homeassistant/components/salus/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
`iot_class` is missing.

<https://developers.home-assistant.io/docs/creating_integration_manifest/#iot-class>